blastoff.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "fmt"
+
+func main() {
+	const Time int = 10
```

Comment:
Start this variable name with a lower case. It doesn't make any difference here, but capital first letters in Go have meaning.